g/grafana/grafana_ubi_8.5.sh

```diff
@@ -0,0 +1,49 @@
+#!/bin/bash -e
+# ----------------------------------------------------------------------------
+# Package          : grafana
+# Version          : v9.1.3
+# Source repo      : https://github.com/grafana/grafana.git
+# Tested on        : UBI 8.5
+# Language         : Go
+# Travis-Check     : True
+# Script License   : Apache License, Version 2 or later
+# Maintainer       : Vinod K <Vinod.K1@ibm.com>
+#
+# Disclaimer       : This script has been tested in root mode on given
+# ==========         platform using the mentioned version of the package.
+#                    It may not work as expected with newer versions of the
+#                    package and/or distribution. In such case, please
+#                    contact "Maintainer" of this script.
+#   
+# ----------------------------------------------------------------------------
+yum -y update
+yum -y install wget git gcc-c++ npm
+
+# Install GO
+wget https://golang.org/dl/go1.18.1.linux-ppc64le.tar.gz
+tar -C /usr/local -xzf go1.18.1.linux-ppc64le.tar.gz
+export PATH=$PATH:/usr/local/go/bin
+
+npm install -g yarn
+
+#Install Node
+yum install -y openssl-devel.ppc64le curl
+curl https://raw.githubusercontent.com/creationix/nvm/master/install.sh | bash
+source ~/.nvm/nvm.sh
+nvm install node
+
+#Build Grafana
+mkdir grafana
+cd grafana
+export GOPATH=`pwd`
+git clone https://github.com/grafana/grafana.git
+cd grafana
+git checkout v9.1.3
+
+go mod verify
+yarn run
```

Comment:
does this both build and run the tests for the node part?

Reply:
This will build the grafana binaries

Reply:
ok, refer to https://github.com/grafana/grafana/blob/main/contribute/developer-guide.md  for how to run the tests as well